libdnf/goal/Goal.cpp

```diff
@@ -1406,12 +1406,20 @@ Goal::Impl::brokenDependencyPkgs(unsigned i)
     return broken_dependency;
 }
 
+bool
+Goal::Impl::isProtectedRunningKernel(Id id)
+{
+    if (DNF_DONT_PROTECT_RUNNING_KERNEL & run_flags)
+        return false;
+    return (id == runningKernelID);
```

Comment:
Minor: Why are you uses brackets behind `return`?

Reply:
Fixed.